### PR TITLE
Support tests with namespaces in Apex Test Sidebar

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Updates the Apex test sidebar so it updates correctly when running test in a project with a namespace.

### What issues does this PR fix or reference?
#1701 

namespace_tests_sidebar

### CODE REVIEW

File: packages/salesforcedx-vscode-apex/test/vscode-integration/views/testDataUtil.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as vscode from 'vscode';
+import { ApexTestMethod } from '../../../src/views/lspConverter';
+
+export function generateApexTestMethod(namespace?: string): ApexTestMethod[] {
```
Comment (lcampos) [inline packages/salesforcedx-vscode-apex/test/vscode-integration/views/testDataUtil.ts]: Moved this out of `testOutline.test.ts` so we can use it in the new tests too.

File: packages/salesforcedx-vscode-apex/test/vscode-integration/views/testOutlineNamespace.test.ts [hunk 2]
```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
```
Comment (lcampos) [inline packages/salesforcedx-vscode-apex/test/vscode-integration/views/testOutlineNamespace.test.ts]: Very similar to the tests in `testOutline.test.ts` but running with data that has a namespace.

File: packages/salesforcedx-vscode-apex/test/vscode-integration/views/testOutlineNamespace.test.ts [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+        expect(testOutline.getHead().children.length).to.equal(1);
+        const testChildGroup = testOutline.getHead().children[0];
+        expect(testChildGroup).instanceof(ApexTestGroupNode);
+        const groupLocation = new vscode.Location(
+          apexNamespacedTestInfo[0].location.uri,
+          APEX_GROUP_RANGE
+        );
+        expect(testChildGroup.location).to.deep.equal(groupLocation);
+        expect(testChildGroup.name).to.equal(
+          apexNamespacedTestInfo[0].definingType
```
Comment (brpowell) [inline packages/salesforcedx-vscode-apex/test/vscode-integration/views/testOutlineNamespace.test.ts]: Does `definingType` include the `tester` namespace?

Comment (brpowell) [reply packages/salesforcedx-vscode-apex/test/vscode-integration/views/testOutlineNamespace.test.ts]: Ah i see it https://github.com/forcedotcom/salesforcedx-vscode/pull/1731/files#diff-c258f087feaf15a9cbc71d6b5c423ad2R16

**Pull request merged**